merge_requests.go

```diff
@@ -697,7 +697,7 @@ func (s *MergeRequestsService) AcceptMergeRequest(pid interface{}, mergeRequest
 	}
 	u := fmt.Sprintf("projects/%s/merge_requests/%d/merge", PathEscape(project), mergeRequest)
 
-	req, err := s.client.NewRequest(http.MethodPut, u, opt, options)
+	req, err := s.client.NewRequest(http.MethodPost, u, opt, options)
```

Comment:
According to the docs, this is really a PUT request. Can you point me to a resource that shows this needs to be a POST request?